src/content/docs/browser/browser-monitoring/browser-pro-features/session-replay.mdx

```diff
@@ -249,6 +249,29 @@ For more details on session replay, see the following sections:
     <link rel="stylesheet" href="assets.yoursite.com/styles.css" crossorigin="anonymous">
     ```
   </Collapser>
+    <Collapser
+        id="Troubleshoot-iframe-content"
+        title="Not seeing replays in your iframes"
+      >
+        ### Problem
+    
+        Session replay feature is enabled, but you're not seeing replay data for iframe content.
+    
+        ### Solution
+    
+        Assuming a web page consists of a top level window with a child iframe, here are some possible fixes:   
+          * If the iframe is **same-origin**,
+            * agent placed in the top level will capture session replay events in both the window and the iframe.
+            * agent placed in the iframe will capture events only in its own context.
+            * agents placed in both window and iframe will each report session replay events independently (with their own session) in line with the two bullets above.
+        * If the iframe is **cross-origin**,
+          * agent placed in the top level will only capture session replay events in the window; iframe will appear blank.
+          * agent placed in the iframe will only capture events in the iframe.
+          * if *recordCrossOriginIframes* is not enabled (current state of agent) in the replay options, agents placed in both window and iframe will each report session replay events independently (with their own session) in line with the two bullets above.
+        * if *recordCrossOriginIframes* <u>is</u> enabled and agents are in both window and iframe, the iframe agent does <u>not</u> report session replay events. However, the window agent will report events with the cross-origin iframe events captured in replay.
```

Comment:
We might want to re-consider adding this to the documentation.  There's not currently a way for customers to configure the `recordCrossOriginIframes` behavior for rrweb.

Reply:
Great, let's remove those two bullet points then.